src/assets/scripts/client/parsers/aircraftCommandParser/aircraftCommandParserMessages.js

```diff
@@ -27,6 +27,7 @@ export const ERROR_MESSAGE = {
     THREE_ARG_LENGTH: `${INVALID_ARG_LENGTH}. Expected exactly three arguments`,
     ZERO_OR_ONE_ARG_LENGTH: `${INVALID_ARG_LENGTH}. Expected zero or one argument`,
     ZERO_TO_THREE_ARG_LENGTH: `${INVALID_ARG_LENGTH}. Expected zero to three arguments`,
+    ZERO_TO_FOUR_ARG_LENGTH: `${INVALID_ARG_LENGTH}. Expected zero to four arguments`,
```

Comment:
Never woulda thought we'd need this lol :+1:

Reply:
me either.  glad it's proving to be flexible though